proto/cosmos/autocli/v1/service.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package cosmos.autocli.v1;
+
+import "cosmos/autocli/v1/options.proto";
+import "cosmos/query/v1/query.proto";
+
+option go_package = "cosmossdk.io/api/cosmos/base/cli/v1;cliv1";
+
+// RemoteInfoService provides clients with the information they need
+// to build dynamically CLI clients for remote chains.
+service RemoteInfoService {
+  // AppOptions returns the autocli options for all of the modules in an app.
+  rpc AppOptions(AppOptionsRequest) returns (AppOptionsResponse) {
+    option (cosmos.query.v1.module_query_safe) = false;
```

Comment:
OK, maybe we can add a proto comment for this annotation why this is set to false explicitly. By scanning quickly, I initially thought it was true (because not omitted), and then I wondered why this particular query was set to false and not others.